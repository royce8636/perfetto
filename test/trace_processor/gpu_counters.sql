select "ts","value","name","gpu_id","description","unit"
from counter
join gpu_counter_track
  on counter.track_id = gpu_counter_track.id
order by "ts";
