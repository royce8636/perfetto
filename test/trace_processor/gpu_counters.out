"counter_id","ts","value","name","ref","ref_type"
1,10,1.000000,"bar",0,"gpu"
2,20,2.000000,"gpu_counter(2)",0,"gpu"
0,30,3.000000,"foo",0,"gpu"
1,40,4.000000,"bar",0,"gpu"
2,50,5.000000,"gpu_counter(2)",0,"gpu"
0,60,6.000000,"foo",0,"gpu"
1,70,7.000000,"bar",0,"gpu"
2,80,8.000000,"gpu_counter(2)",0,"gpu"
0,90,9.000000,"foo",0,"gpu"
